Documentation/diagnostics/source_link.md

```diff
@@ -133,6 +190,7 @@ In this example:
             }
         }
         ```
+    - Starting in [VS 2017 version 15.7 Preview 3](https://blogs.msdn.microsoft.com/visualstudio/2018/04/09/visual-studio-2017-version-15-7-preview-3/), the debugger can automatically authenticate source link requests for VSTS and GitHub repositories.
 - [C# Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
     - Source Link is a planned feature but is not yet supported. 
```

Comment:
Can you fix this part also while you are updating? It should say: "Supports Source Link as of C# extension version 1.15"